src/pages/forms/[id].tsx

```diff
@@ -152,3 +157,7 @@ function respectNewLines(text: string | null | undefined) {
     </span>
   ));
 }
+
+async function closeForm(formId: string) {
+  // TODO: Call API endpoint to update form's is_open status
```

Comment:
@ellipsis-dev looks good, but we actually need to update the row in Supabase. There's an `is_open` column that needs to flip